HacktoberfestProject.Web/wwwroot/css/output.css

```diff
@@ -18397,11 +18397,13 @@ video {
 }
```

Comment:
Note: I didn't actually edit this file, but I did run `npm run tailwind` to rebuild which generated these changes prior to me pushing. If this was incorrect, let me know and I'll revert the change 😃